src/xebia_email_signature/templates/signature.html.jinja

```diff
@@ -6,28 +6,28 @@
     <table summary="contact details">
         <tr style="line-height: 1;">
         {% if data.get('profile_picture') %}
-        <td>
+        <td style="width: 60px; text-align: center;">
```

Comment:
Email clients drop styles on copy, pasting and forwarding emails. The image must be the expected width and height.

Reply:
Added